libs/zedUpload/ociutil/oci.go

```diff
@@ -127,15 +126,11 @@ func PullBlob(registry, repo, hash, localFile, username, apiKey string, maxsize
 		}
 	}
 
-	// if we have only a tag, we know it is a manifest
-	if _, ok := ref.(name.Tag); ok {
-		logrus.Infof("PullBlob: requested manifest or had tag without hash, so just pulling root for %s", image)
-		r, contentType, err = ociGetManifest(ref, opts)
-		if err != nil {
-			return 0, "", err
-		}
-	} else {
-		// we had a hash, so get the actual layer, but fall back to manifest
+	// check if we have a manifest
+	r, contentType, err = ociGetManifest(ref, opts)
+	if err != nil {
+		logrus.Infof("PullBlob(%s): unable to fetch manifest. %s", image, err.Error())
```

Comment:
With the change, this log line is confusing, and will look like an error. Might want to add a line before like, "trying to get <ref> as manifest", and then this line can be, "unable to fetch <ref> as manifest, trying as blob", etc.